packages/service-library/src/servicelib/progress_bar.py

```diff
@@ -35,9 +39,12 @@ def _normalize_weights(steps: int, weights: list[float]) -> list[float]:
 
 
 @dataclass(slots=True, kw_only=True)
-class ProgressBarData:
+class ProgressBarData:  # pylint: disable=too-many-instance-attributes
```

Comment:
Correct me if I am wrong, but I think we try to put the pylint exceptions at the top of the file. Maybe this is only in the tests 🤔. I quite like the convention, so I would suggest to do it here as well.